build-system/pr-check/visual-diff-tests.js

```diff
@@ -56,9 +56,9 @@ function main() {
     ) {
       downloadDistOutput(FILENAME);
       timedExecOrDie('gulp update-packages');
-      timedExecOrDie('gulp visual-diff --nobuild');
+      timedExecOrDie('gulp visual-diff --compiled --nobuild');
     } else {
-      timedExecOrDie('gulp visual-diff --nobuild --empty');
+      timedExecOrDie('gulp visual-diff --empty');
```

Comment:
why remove `--nobuild` here?

Reply:
When you pass in `--empty` to `gulp visual-diff`, all other build-related flags are ignored, so `--nobuild` was unnecessary in the first place.

Also, this is the only place where `--nobuild` was removed.